# The XLA BUILD file for zlib doesn't export zconf.h, which is a public header.
--- third_party/zlib.BUILD
+++ third_party/zlib.BUILD
@@ -1,10 +1,10 @@
@@ -34 +34 @@
-    hdrs = ["zlib.h"],
+    hdrs = ["zconf.h", "zlib.h"],
